cost/turbonomics/credential_refresh/turbonomic_cred_refresh.pt

```diff
@@ -0,0 +1,167 @@
+name "Turbonomic Credential Refresh"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that refreshes the cookie used to authenticate with Turbonomic APIs"
+severity "low"
+category "Operational"
+default_frequency "hourly"
+info(
+  version: "0.1",
+  provider: "",
+  service: "",
+  policy_set: "",
+	publish: "false"
+)
+
+parameter "param_turbonomic_username" do
+  type "string"
+  label "Turbonomic Username"
+  description "The username used to authenticate with Turbonomic APIs"
+end
+
+parameter "param_turbonomic_password" do
+  type "string"
+  no_echo true
+  label "Turbonomic Password"
+  description "The password used to authenticate with Turbonomic APIs"
+end
+
+parameter "param_turbonomic_host" do
+  type "string"
+  label "Turbonomic Host"
+  description "Your Turbonomic host or IP."
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+credentials "auth_flexera" do
+  schemes "oauth2"
+  label "flexera"
+  description "Select Flexera One OAuth2 credentials"
+  tags "provider=flexera"
+end
+
+###############################################################################
+# Datasources & Scripts
+###############################################################################
+
+# Retrieves all applied policies in the project where this policy is applied
+datasource "ds_applied_policies" do
+  request do
+    auth $auth_flexera
+    host rs_governance_host
+    path join(["/api/governance/projects/", rs_project_id, "/applied_policies"], "")
+    header "Api-Version", "1.0"
+  end
+  result do
+    encoding "json"
+    collect jq(response, ".items[]") do
+      field "id", val(col_item, "id")
+      field "name", val(col_item, "name")
+      field "info_source", jq(col_item, ".info.source")
+    end
+  end
+end
+
+# Filters the applied policy where the info.source is set to either "Turbonomic" or "Turbonomics"
+datasource "ds_turbo_applied_policies" do
+  run_script $js_filter_turbo_policies, $ds_applied_policies
+end
+
+script "js_filter_turbo_policies", type: "javascript" do
+  parameters "applied_policies"
+  result "filtered_policies"
+  code <<-EOS
+  var filtered_policies = _.map(_.filter(applied_policies, function(applied_policy) {
+    return applied_policy.info_source == "Turbonomics" || applied_policy.info_source == "Turbonomic"
+  }), function(policy){policy.time = Date.now(); return policy});
+  EOS
+end
+
+###############################################################################
```

Comment:
Policy and Escalations should be separate sections in the policy. See:

https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Basic-Policy-Structure

Reply:
Thank you, done